src/components/Layout/PageContent.tsx

```diff
@@ -1,17 +1,17 @@
-import React, { FC } from 'react';
+import React, { FC, ReactNode } from 'react';
 import { Typography } from 'antd';
-
 import style from './PageContent.module.scss';
+
 const { Title } = Typography;
 
 type PageContentProps = {
-  title: string | void;
+  title: string | ReactNode;
 };
 
 const PageContent: FC<PageContentProps> = ({ children, title }) => (
   <div className={style.pageContentContainer}>
     <div className={style.pageContent}>
-      {title && <Title level={1}>{title}</Title>}
+      {!!title && typeof title === 'string' ? <Title level={1}>{title}</Title> : title}
```

Comment:
@francisl here it's just a quick fix since for the page title I needed more than a string. Surely improvements can be made.

Reply:
`!!` not needed,